waku/waku_filter_v2/common.nim

```diff
@@ -13,11 +13,11 @@ const
 type
   FilterSubscribeErrorKind* {.pure.} = enum
     UNKNOWN = uint32(000)
-    PEER_DIAL_FAILURE = uint32(200) # TODO shouldn't this be an error code, e.g. 504 Gateway Timeout?
```

Comment:
There are a couples of TODOs in the code base and I always think to myself. Why not just change it instead of adding a TODO your touching this part of the code anyway. If you need to remember to do something later should open an issue IMO.